sklearn/metrics/_ranking.py

```diff
@@ -396,8 +396,9 @@ class scores must correspond to the order of ``labels``,
 
     average : {'micro', 'macro', 'samples', 'weighted'} or None, \
             default='macro'
-        If ``None``, the scores for each class are returned. Otherwise,
-        this determines the type of averaging performed on the data:
+        If ``None``, the scores for each class are returned (only implemented
+        for ``multi_class='ovr'``). Otherwise, this determines the type of
+        averaging performed on the data:
         Note: multiclass ROC AUC currently only handles the 'macro' and
         'weighted' averages.
 
```

Comment:
To make this point clearer, we can move the docstring closer to the multiclass part:

```suggestion
        If ``None``, the scores for each class are returned (only implemented
        for ``multi_class='ovr'``). Otherwise, this determines the type of
        averaging performed on the data:
        Note: multiclass ROC AUC currently only handles the 'macro' and
        'weighted' averages. For multiclass targets, `average=None`
        is only implemented for `multi_class='ovo'`.
```